test/controllers/slides_controller_test.rb

```diff
@@ -233,4 +233,35 @@ class SlidesControllerTest < ActionController::TestCase
     assert_select "video[controls]"
     assert_select "video[muted]"
   end
+
+  # This admission test is to confirm that the admission options are correct, due to
+  # previously being swapped with audience options.
+  test "check for proper options for admission labels on scheduled slides" do
+    # Arrange
+    sign_in users(:super_admin)
+    slide = scheduled_items(:free_student_event)
+
+    # Act
+    get :preview, id: @slide
+    assert_response :success
+
+    # Assert
+    assert_equal "Free", slide.admission
+  end
+
+  # This audience test is to confirm that the audience options are correct, due to
+  # previously being swapped with admission options.
+  test "check for proper options for audience labels on scheduled slides" do
+    # Arrange
+    sign_in users(:super_admin)
+    slide = scheduled_items(:free_student_event)
+
+    # Act
+    get :preview, id: @slide
+
+    assert_response :success
+
+    # Assert
+    assert_equal "Students", slide.audience
+  end
```

Comment:
Combine these into one test and reference the Github issue. Also you're not really proving that the issue as been resolved. You are only showing that the fixture was created with the expected field value. A test that address the defect will use something like [assert_select](https://apidock.com/rails/ActionController/Assertions/SelectorAssertions/assert_select) to verify that field label and options are no longer mixed up.